ONSdigital/eq-terraform
# Add support for google analytics

**_What**_

Adds support for google analytics

**_How to test**_

1) Checkout the branch
2) add the var `google_analytics_code` to terraform.tfvars and set it to the GA Tracking Code
3) Deply the environment and verify that the tracking code is installed in the survey-runner

**_Who can test**_

Anybody except @weapdiv-david
